tests/test_operator.py

```diff
@@ -203,6 +203,92 @@ def test_arguments(self):
         op4 = switchconfig(language='openmp')(Operator)(Eq(u, u + 1), language='C')
         assert '#pragma omp for' not in str(op4)
 
+    def test_nested_lowering_and_indexing(self):
```

Comment:
can you move all of these tests right below "test_relationals_index_shifting" ?

Reply:
can you drop "and_indexing" from the name? as I said a few times already, indexification is part (a sub-pass) of lowering

Reply:
and to make it more complete, can you use `eq = Eq(..., subdomain=grid.inner)` so we test the whole lowering